lib/matplotlib/transforms.py

```diff
@@ -1831,7 +1831,7 @@ def __init__(self, matrix=None, **kwargs):
         if matrix is None:
             # A bit faster than np.identity(3).
             matrix = IdentityTransform._mtx.copy()
-        self._mtx = matrix
+        self._mtx = matrix.copy()
```

Comment:
While I see why this may be needed, this may be quite costly too, and I think we may want to check that this doesn't slow down "global" usage (e.g., drawing a figure with plenty of subaxes) more than the speedup gained by skipping numpy below.

Reply:
Before the PR, every use of the transformation did a copy. Note that the replaced dot products don't use a out parameter. I would also argue that the copy is the correct thing to do, since using a view as a parameter is a seldom used pattern.

Reply:
Ah yes, I guess you pay here once what you used to pay a lot of times before...